src/VirtualList.jsx

```diff
@@ -337,7 +337,10 @@ class VirtualList extends React.Component {
     const { winStart, winSize, avgRowHeight } = this.state;
     const winEnd = Math.min(items.length - 1, winStart + winSize - 1);
     const paddingTop = winStart * avgRowHeight;
-    const paddingBottom = (items.length - winStart - winSize) * avgRowHeight;
+    let paddingBottom = 0;
+    if ((items.length - winStart - winSize) >= 0) {
+       paddingBottom = (items.length - winStart - winSize) * avgRowHeight;
+    }
```

Comment:
I think it would be more consistent with the rest of the code to write it like this:

```javascript
const paddingBottom = Math.max((items.length - winStart - winSize) * avgRowHeight, 0);
```